xyzservices/data/providers.json

```diff
@@ -2989,5 +2989,80 @@
             "html_attribution": "Map tiles by <a href=\"https://labs.strava.com/heatmap\">Strava 2021</a>",
             "name": "Strava.Winter"
         }
+    },
+    "SwissFederalGeoportal": {
+        "NationalMapColor": {
+            "url": "https://wmts.geo.admin.ch/1.0.0/ch.swisstopo.pixelkarte-farbe/default/current/3857/{z}/{x}/{y}.jpeg",
+            "html_attribution": "<a target=\"_blank\" href=\"https://www.swisstopo.admin.ch/\">swisstopo</a>",
+            "attribution": "© swisstopo",
+            "bounds": [
+                [
+                    5.140242,
+                    45.398181
+                ],
+                [
+                    11.47757,
+                    48.230651
```

Comment:
The order of bounds is opposite. It should be lat, lon. You can check the other Swiss dataset on the top of the file.